### DISCUSSION TITLE
Navigation bar

### DISCUSSION THREAD

**Discussion by venom6191991** [Security-Onion-Solutions/securityonion#8121] [Unsupported Versions] [answered]
Does anyone know how to add a new button under the overview/Alert/Hunt section? I just want to create a new button that executes a local script

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/soc-customization.html#links

**Reply by venom6191991** [reply]
Those are links for the tools section, not the items on top, IE the overview, alerts hunt, cases. I would like to add a button in that section.

**Reply by dougburks** [reply]
The links at the top of the navigation bar are links to apps that exist within Security Onion Console. The only way to add a link in that section would be to manually modify the source code and build your own Docker image and that's not something that we support.
